relm4/Cargo.toml

```diff
@@ -27,6 +27,7 @@ adw = { git = "https://gitlab.gnome.org/World/Rust/libadwaita-rs", optional = tr
 
 async-broadcast = "0.3.4"
 async-oneshot = "0.5.0"
+derivative = "2.2.0"
```

Comment:
I see why it's useful, but I don't want to pull in a dependency that has more lines of code than the relm4 crate itself if implementing the traits manually is an option.

Reply:
Fair, I'll post a new revision with manual impls.